src/models/converters/data_decoded.rs

```diff
@@ -0,0 +1,58 @@
+use crate::models::commons::DataDecoded;
+use crate::models::service::transactions::SettingsInfo;
+
+impl DataDecoded {
+    pub(super) fn to_settings_info(&self) -> SettingsInfo {
+        match self.method.as_str() {
+            "setFallbackHandler" => {
```

Comment:
maybe we should make these constrants in the utils module and it would be nice to only have 1 place were we manage these string: https://github.com/gnosis/safe-client-gateway/blob/c21802dac969ad3a59b247dcd9548ef9dc6d359a/src/utils/mod.rs#L14

Reply:
beware of the crap that I was forced to introduce because of  how the syntax for `match` arm works xD